csrc/device_lower/pass/expr_sort.cpp

```diff
@@ -1515,9 +1515,8 @@ void ExprSegmentationSorter::sort() {
       fusion_->outputs().end(),
       std::back_inserter(non_pointer_arithmetic_outs),
       [this](Val* out) {
-        auto [in, alias_info] = fusion_->getOutputAlias(out);
-        return in == nullptr ||
-            alias_info->type != AliasType::PointerArithmetic;
+        return fusion_->getOutputAlias(out).type !=
+            AliasType::PointerArithmetic;
```

Comment:
Do we need the check `fusion_->getOutputAlias(out).aliased_io != nullptr`?

Reply:
No, `PointerArithmetic` must have a non-null `aliased_io` so a check like that is redundant.